go/vt/tabletserver/schema_info.go

```diff
@@ -291,29 +312,24 @@ func (si *SchemaInfo) mysqlTime(ctx context.Context) int64 {
 	return t
 }
 
-// safe to call this if Close has been called, as si.ticks will be stopped
-// and won't fire
-func (si *SchemaInfo) triggerReload() {
-	si.ticks.Trigger()
-}
-
 // ClearQueryPlanCache should be called if query plan cache is potentially obsolete
 func (si *SchemaInfo) ClearQueryPlanCache() {
 	si.queries.Clear()
 }
 
 // CreateOrUpdateTable must be called if a DDL was applied to that table.
-func (si *SchemaInfo) CreateOrUpdateTable(ctx context.Context, tableName string) {
+func (si *SchemaInfo) CreateOrUpdateTable(ctx context.Context, tableName string) error {
```

Comment:
I'm thinking we can get rid of both functions and just call ReloadSchema instead.